clouddq/main.py

```diff
@@ -230,7 +223,6 @@ def main(  # noqa: C901
     print_sql_queries: bool = False,
     skip_sql_validation: bool = False,
     summary_to_stdout: bool = False,
-    enable_experimental_bigquery_entity_uris: bool = False,
```

Comment:
incorporated the change

Reply:
We should probably also

1. update the flag description (e.g. it's always true, please deprecated the usage)
2. emit a warning log if we find that somebody is still using the flag.

I think ultimately we should find a way to remove it.

Reply:
Let's update the flag description and hide it at least.

We should consider ignoring unknown flags in the future.

Reply:
I don't think we need to keep this parameter for main method, do we? It's not supposed to be used anywhere in the code.

To avoid breaking existing scripts, the only thing we need to keep it the flag/option definition.